src/ralph/admin/filters.py

```diff
@@ -187,6 +187,36 @@ def choices(self, cl):
         }
 
 
+class DecimalListFilter(DateListFilter):
+
+    """Renders filter form with decimal field."""
+
+    template = 'admin/filters/decimal_filter.html'
+
+    def queryset(self, request, queryset):
+        if any(self.value()):
+            value = self.value()
+            if value[0]:
+                queryset = queryset.filter(**{
+                    '{}__gte'.format(self.field_path): value[0]
+                })
+            if value[1]:
+                queryset = queryset.filter(**{
+                    '{}__lte'.format(self.field_path): value[1]
+                })
+
+            return queryset
+
+    def choices(self, cl):
```

Comment:
do we have to use choices here (and in date filter too)? I don't like this yielding-only-single-element.